### PR TITLE
Dash daq satellite dashboard

### PR DESCRIPTION
**Pull request by YunkeXiao**
Issue for app: #53 

# App pull request
- [x] This is a new app
- [ ] I am improving an existing app (redesigns/code "makeovers")

## About

- Playground deployment URL (new version): https://dash-playground.plotly.host/dash-daq-satellite-dashboard/

## Workflow

- [x] I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [x] If the app is a redesigned and/or restyled version of an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.
- [x] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.
- [x] If the app is a reimplementation of a Python gallery app for the
  DashR gallery, the app in this PR mimics, as closely as possible,
  the style and functionality of the existing app.=
- [x] I have removed *all Google Analytics code* from the app's
  `assets/` folder.

## The pre-review review

I have addressed all of the following questions:

- [x] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.)
- [x] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)]
- [x] Am I reinventing the wheel? (I have used appropriate packages to
  lessen the volume of code that needs to be maintained.)

### CODE REVIEW

File: apps/dash-daq-satellite-dashboard/data/gps_data.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+from random import randint
```
Comment (celinehuang) [inline apps/dash-daq-satellite-dashboard/data/gps_data.py]: remove unused imports!

File: apps/dash-daq-satellite-dashboard/app.py [hunk 2]
```diff
@@ -0,0 +1,1064 @@
...
+)
+
+# This is for gunicorn
+server = app.server
+
+# Mapbox
+MAPBOX_ACCESS_TOKEN = "pk.eyJ1IjoicGxvdGx5bWFwYm94IiwiYSI6ImNqdnBvNDMyaTAxYzkzeW5ubWdpZ2VjbmMifQ.TXcBE-xg9BFdV2ocecc_7g"
+MAPBOX_STYLE = "mapbox://styles/plotlymapbox/cjyivwt3i014a1dpejm5r7dwr"
+
+##############################################################################################################
```
Comment (shammamah) [inline apps/dash-daq-satellite-dashboard/app.py]: You can get rid of these comments!

File: apps/dash-daq-satellite-dashboard/data/gps_data_h.csv [hunk 3]
```diff
@@ -0,0 +1,61 @@
+lat,lon
```
Comment (shammamah) [inline apps/dash-daq-satellite-dashboard/data/gps_data_h.csv]: Why is this file full of zeroes?

File: apps/dash-daq-satellite-dashboard/data/non_gps_data_h.csv [hunk 4]
```diff
@@ -0,0 +1,61 @@
+elevation,temperature,speed,fuel,battery
```
Comment (shammamah) [inline apps/dash-daq-satellite-dashboard/data/non_gps_data_h.csv]: This file is also full of zeroes

File: apps/dash-daq-satellite-dashboard/data/gps_data_h.csv (hunk 3 above)
Comment (YunkeXiao) [reply apps/dash-daq-satellite-dashboard/data/gps_data_h.csv]: Need if for default values

Comment (YunkeXiao) [reply apps/dash-daq-satellite-dashboard/data/gps_data_h.csv]: Need it for default values

File: apps/dash-daq-satellite-dashboard/data/non_gps_data_h.csv (hunk 4 above)
Comment (YunkeXiao) [reply apps/dash-daq-satellite-dashboard/data/non_gps_data_h.csv]: Need it for default values

File: apps/dash-daq-satellite-dashboard/data/gps_data_h.csv (hunk 3 above)
Comment (shammamah) [reply apps/dash-daq-satellite-dashboard/data/gps_data_h.csv]: Could you instead generate this from within the Python file?

**Pull request merged**